### PR TITLE
NEXTEUROPA-5459 : Remove all third party code from the ec_resp theme

### PR DESCRIPTION
**Pull request by senseca**

### CODE REVIEW

File: resources/multisite_drupal_standard.make [hunk 1]
```diff
@@ -617,6 +617,39 @@ libraries[Leaflet.widget][destination] = "libraries"
...
 libraries[Leaflet.widget][download][url] = "https://github.com/tnightingale/Leaflet.widget.git"
 
+; Ec_resp theme : Bootstrap 3.3.5
+libraries[ec_resp_bootstrap][download][type] = get
+libraries[ec_resp_bootstrap][download][url] = https://github.com/twbs/bootstrap/releases/download/v3.3.5/bootstrap-3.3.5-dist.zip
+libraries[ec_resp_bootstrap][download][file_type] = "zip"
+libraries[ec_resp_bootstrap][destination] =  "themes/ec_resp"
+libraries[ec_resp_bootstrap][directory_name] = bootstrap
+
+libraries[ec_resp_bootstrap_less][destination] = "libraries"
```
Comment (gervasek) [inline resources/multisite_drupal_standard.make]: useless, destination is defined below

**Comment by gervasek** [comment]
The latest test of this branch is marked as failed, but no error appears on logs: https://continuousphp.com/git-hub/ec-europa/platform-dev/build/4307556a-d8e5-43cd-acff-81400064317c/output

**Comment by ppaulis** [comment]
@gervasek Hello!

we are very sorry for this interruption! We made a deployment of our workers this afternoon to restart failed or timed out activities. In the future, this should prevent the problem you experienced half an hour ago. You can restart the build by pushing a commit to the same branch or we can restart the builds for you if you prefer.

**Comment by gervasek** [comment]
CPHP tests still failed: https://continuousphp.com/git-hub/ec-europa/platform-dev/build/235971a7-7c13-4c21-910e-404d9dc436cd/output

**Comment by ppaulis** [comment]
@gervasek I took a closer look at this build. Both Behat activities reported, that they have finished their task and that's a bit strange. I'll get back to you, as soon as I have more infos.

**Pull request merged**